src/kv/source.rs

```diff
@@ -1,25 +1,65 @@
-//! Sources for key-value pairs.
+//! Sources for key-values.
+//!
+//! This module defines the [`Source`] type and supporting APIs for
+//! working with collections of key-values.
 
 use crate::kv::{Error, Key, ToKey, ToValue, Value};
 use std::fmt;
 
-/// A source of key-value pairs.
+/// A source of key-values.
 ///
 /// The source may be a single pair, a set of pairs, or a filter over a set of pairs.
-/// Use the [`Visitor`](trait.Visitor.html) trait to inspect the structured data
+/// Use the [`VisitSource`](trait.VisitSource.html) trait to inspect the structured data
```

Comment:
Nit: don't think it's needed any more. But if we have it in other places, then let's leave it and do it some other time for all links.
```suggestion
/// Use the [`VisitSource`] trait to inspect the structured data
```

Reply:
Good call. I think I still need to do a pass over links and make sure they're all resolved correctly so will simplify them wherever possible.